internal/nodes/ingress_model_rel.go

```diff
@@ -291,7 +292,12 @@ func IngressChanges(ingName string, namespace string, key string) ([]string, boo
 		if k8serrors.IsNotFound(err) {
 			// Remove all the Ingress to Services mapping.
 			// Remove the references of this ingress from the Services
-			objects.SharedSvcLister().IngressMappings(namespace).RemoveIngressMappings(ingName)
+			svcToDel := objects.SharedSvcLister().IngressMappings(namespace).RemoveIngressMappings(ingName)
```

Comment:
this can move inside the conditional below

Reply:
We need to remove the mapping even if NPL is not enabled, so the function call has to be made, the return value can be ignored.